components/Carousel.tsx

```diff
@@ -0,0 +1,49 @@
+import { FunctionComponent, useState } from "react";
+import Button from "./Button";
+import CarouselPagination from "./CarouselPagination";
+import ICarousel from "types/CarouselProps";
+
+const carouselContent = [

```

Comment:
Isso daqui tem que estar saindo do storybook, não aqui no componente

Reply:
Maybe yes. I think that this can stay in the component since it's static files (images).